Beta_Integrations/ProofpointThreatResponse/ProofpointThreatResponse.yml

```diff
@@ -1,134 +1,892 @@
+      name: comments
+      required: true
+      secret: false
+    deprecated: false
+    description: Adds comments to an existing Threat Response incident, by incident
+      ID.
+    execution: false
+    name: proofpoint-tr-update-incident-comment
+    outputs:
+    - contextPath: ProofPointTRAP.IncidentComment.id
+      description: The ID of the comment.
+      type: Number
+    - contextPath: ProofPointTRAP.IncidentComment.incident_id
+      description: The ID of the incident.
+      type: Number
+    - contextPath: ProofPointTRAP.IncidentComment.response_id
+      description: The ID of the response.
+      type: Number
+    - contextPath: ProofPointTRAP.IncidentComment.user_id
+      description: The ID of the user.
+      type: String
+    - contextPath: ProofPointTRAP.IncidentComment.history_type
+      description: The history type.
+      type: String
+    - contextPath: ProofPointTRAP.IncidentComment.state_from
+      description: The state from of the incident.
+      type: String
+    - contextPath: ProofPointTRAP.IncidentComment.state_to
+      description: The state to of the incident.
+      type: String
+    - contextPath: ProofPointTRAP.IncidentComment.summary
+      description: The summary of the comments.
+      type: String
+    - contextPath: ProofPointTRAP.IncidentComment.detail
+      description: The details of the comment.
+      type: String
+    - contextPath: ProofPointTRAP.IncidentComment.created_at
+      description: The date that the incident was created.
+      type: Date
+    - contextPath: ProofPointTRAP.IncidentComment.updated_at
+      description: The date that the incident was last udpated.
+      type: Date
+  - arguments:
+    - default: false
+      description: The ID value of the incident to add the user to.
+      isArray: false
+      name: incident_id
+      required: true
+      secret: false
+    - default: false
+      description: The list of targets to add to the incident.
       isArray: true
-    - name: expiration
-      description: Expiration of the URLs
-    description: Block URL
-  - name: proofpoint-tr-block-hash
-    arguments:
-    - name: hash
-      required: true
-      description: List of hashes
+      name: targets
+      required: true
+      secret: false
+    - default: false
+      description: The list of attackers to add to the incident.
       isArray: true
-    - name: expiration
-      description: Expiration of the hash
-    description: Block hash
+      name: attackers
+      required: true
+      secret: false
+    deprecated: false
+    description: Assigns a user to an incident as a target or attacker.
+    execution: false
+    name: proofpoint-tr-add-user-to-incident
+  - arguments:
+    - default: false
+      description: 'POST URL of the JSON alert source. You can find it by navigating
+        to Sources -> JSON event source -> POST URL '
+      isArray: false
+      name: json_source_id
+      required: true
+      secret: false
```

Comment:
They will have to go to the UI to get that? we should have a solution for this in demisto, they should not go to the product to use it